src/test/test-memfd-util.c

```diff
@@ -11,21 +12,46 @@
 
 TEST(memfd_get_sealed) {
 #define TEST_TEXT "this is some random test text we are going to write to a memfd"
-        _cleanup_close_ int fd = -EBADF;
-
-        fd = memfd_new_full("test-memfd-get-sealed", MFD_ALLOW_SEALING);
+        _cleanup_close_ int fd = memfd_new_full("test-memfd-get-sealed", MFD_ALLOW_SEALING);
         if (fd < 0) {
-                assert_se(ERRNO_IS_NOT_SUPPORTED(fd));
+                ASSERT_TRUE(ERRNO_IS_NOT_SUPPORTED(fd));
                 return;
         }
+        if (ERRNO_IS_NOT_SUPPORTED(fd))
+                return (void) log_tests_skipped_errno(fd, "Could not make new %s : %m", "test-memfd-get-sealed");
+        ASSERT_OK(fd);
```

Comment:
Please drop the second `if` block, move log_tests_skipped_errno() to the first `if` block, drop `%s` and `%m` in the log message, and drop redundant ASSERT_OK() check after `if`:
```c
if (fd < 0) {
        ASSERT_TRUE(ERRNO_IS_NOT_SUPPORTED(fd));
        return (void) log_tests_skipped_errno(fd, "Failed to create new memfd");
}
```